notebooks/official/experiments/comparing_local_trained_models.ipynb

```diff
@@ -32,51 +32,48 @@
         "<table align=\"left\">\n",
```

Comment:
<p>Separated this into 4 cells.</p><p>Move dataset after Objective.</p><p> </p><p>Add "Vertex AI TensorBoard" to bullet list of billable components.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/971/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>done</p>